mobile/Screen/MapScreen.js

```diff
@@ -193,10 +203,4 @@ function MapScreen({ navigation }) {
   );
 }
 
-MapScreen.propTypes = {
-  navigation: PropTypes.shape({
-    navigate: PropTypes.func,
-  }).isRequired,
-};
-
```

Comment:
I think this might have been an accidental deletion because it causes errors later on